src/sql/platform/connection/common/connectionManagementService.ts

```diff
@@ -693,7 +693,10 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 	}
 
 	public hasRegisteredServers(): boolean {
-		return this.doHasRegisteredServers(this.getConnectionGroups());
+		const groups: ConnectionProfileGroup[] = this.getConnectionGroups();
+		const ret: boolean = this.doHasRegisteredServers(groups);
```

Comment:
minor: it'd be nice to name this `hasServers` instead of the undescriptive `ret`